tests/e2e/cucumber/publicLinkToFolderJourney.feature

```diff
@@ -21,3 +21,8 @@ Feature: Create public link to folder
       | resource     | name         | role     | dateOfExpiration | password |
       | folderPublic | myPublicLink | uploader | +5 days          | 12345    |
     Then "Alice" should see 1 public link
+    When the unauthorized user access the last public link created
+    And the unauthorized user authenticates with correct password "12345" to access public link
```

Comment:
```suggestion
    And the public accesses the last created public link with the correct password "12345"
```

We have a step in oC10 core:
```
When the public accesses the last created public link with password "pass123" using the webUI
```
That step both browses to the link and enters the password, so that covers 2 of the steps here in 1 step. Maybe we can write something like that?

Reply:
Combining the two steps is good. I would suggest something like `When the unauthorized user opens the last created public link with password "pass123"` Then he sees something there. 
It sounds clearer to me